src/vm/amd64/unixstubs.cpp

```diff
@@ -72,18 +72,6 @@ extern "C"
         return eax;
     }
 
-    DWORD xmmYmmStateSupport()
```

Comment:
Please keep it here too instead using the version from PAL. Otherwise, to keep PAL konventions, you would need to make it a full blown PAL function, exposed by the pal.h header etc. 
This is so lightweight that it is not worth expanding the already too large PAL surface.